lib/ClangImporter/ImportType.cpp

```diff
@@ -151,14 +151,18 @@ namespace {
   struct ImportResult {
     Type AbstractType;
     ImportHint Hint;
+    bool incompleteType; // true when the type has only been forward declared in
```

Comment:
Let's keep with the coding style around the code we add… sadly we will have to use `UpperCamelCase` for member variables.